### ISSUE TITLE
New handicap

### ISSUE THREAD

**Issue by tyskodima** [online-go/online-go.com#909]
It would be good to have an opportunity to select a type of handicap for tournaments: stones or komi. For example, if 1k and 9k play a tournament game, the handicap is 7 stones. It would be good it tournament creator will set the coefficient for the handicap (for example 10) and then instead of 7 stones of handicap, 9k player will have 70.5 points of komi.

My city has a small Go community and the difference between the levels of players is large. It is not interesting for high-level players to play almost all games with 9 stones of handicap.

**Comment by BHydden** [comment]
While I don't personally prefer this method, many people at my local club do and I think looking into implementing this is worthwhile (depending on how difficult adding an extra handicap option is).

**Comment by KristiSik** [comment]
@tyskodima agree with you. I think it would be a really nice feature.

**Comment by SashaPshenychniy** [comment]
It would be nice feature to have. However having coefficient for the handicap configurable may influence established ratings if such games will be counted as ranked, so it should be considered carefully.

**Comment by prorrad** [comment]
It would be nice to give it a try at least for unrated games.

**Comment by kobakitty** [comment]
Yes, its a good idea! I would personally see this done by reducing whites komi into negative points, like 6 points per each rank difference and have the stronger player take white like normally, but making the weaker player white with huge komi would be ok too. As long as this would be automated feature, so that we don't have to manually calculate it before sending the challenge, this would be fantastic! ^____^

**Comment by flovo** [comment]
I want to add that if you adjust komi, there is no reason for the stronger player to take white but tradition.

**Comment by BHydden** [comment]
> but tradition.

Long live tradition! 😃

**Comment by kevincoleman** [comment]
Just found this after setting up a game with a 2k friend who stomps me in handicap games. I set up a game where he plays black and I get a ton of komi. It’s definitely playing more naturally for me (we’ve got [a very close game](https://online-go.com/game/31156263) this way).

While I agree that tradition is the only reason to prefer one color over another, it would have been more natural to set a negative komi and run with it. As it was I looked up [a chart on Sensei’s Library](https://senseis.xmp.net/?ReverseKomiTable) and added my normal komi to it. It did feel a little imposing to ask a player 8 ranks higher than me to play black just for the handicap style.

I have no idea if we’d even want to add these options for rated games, but I can even see an eventuality where players can set a preferred handicap type in their profiles and the match-maker just figures out how to assign points/stones.

**Comment by BHydden** [comment]
I still think this is worth adding, especially for smaller boards where a whole stone is a huge handicap.

**Comment by JQStitzel** [comment]
I still think this is worth looking into

**Comment by anoek** [comment]
Indeed, this touches a bit on some of the work @dexonsmith has been doing with the ratings work, so it's certainly worth keeping open.
